DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc

```diff
@@ -1276,6 +1281,13 @@ bool SiStripMonitorTrack::clusterInfos(
       if(noise == 0.0)
 	LogDebug("SiStripMonitorTrack") << "Module " << detid << " in Event " << eventNb << " noise " << noise << std::endl;
     }
+    if (TkHistoMap_On_ && (flag == OnTrack)) {
```

Comment:
otherwise, please change this (and the same just below) to If ( TkHistoMap_On_ != nullptr && ...)